content/doc/applications/docker/_index.md

```diff
@@ -35,15 +35,172 @@ Clever Cloud allows you to deploy any application running inside a Docker contai
 
  {{% content/set-env-vars %}}
 
-{{< readfile file="language-specific-deploy/docker.md" >}}
+## Configure your Docker application
 
- {{% content/env-injection %}}
+### Mandatory configuration
 
- {{% content/deploy-git %}}
+Be sure that you:
 
- {{% content/link-addon %}}
+* push on the **master branch**.
+* have and commit a file named **Dockerfile** or use the **CC_DOCKERFILE** [environment variable]({{< ref "doc/reference/reference-environment-variables.md#docker" >}}) if your Dockerfile has a different name, [Here is what it will look like](https://docs.docker.com/develop/develop-images/dockerfile_best-practices "Dockerfile").
+* run the application with `CMD` or `ENTRYPOINT` in your Dockerfile.
+* listen on HTTP **port 8080** by default (you can set your own port using `CC_DOCKER_EXPOSED_HTTP_PORT=<port>` environment variable).
 
+### Dockerfile contents
+
+You can virtually put everything you want in your Dockerfile. The only mandatory (for us) instruction to put in it is:
+
+```bash
+CMD <command to run>
+```
+
+**command to run**: this is the command that starts your application. Your application **must** listen on port 8080. It can be easier for you to put a script in your docker image and call it with the CMD instruction.
+
+### Memory usage during building
+
+If the building step of your app crashes because it uses more memory that it's available, you'll have to split the building and running steps and enable [Dedicated build instance]({{< ref "doc/administrate/apps-management.md#edit-application-configuration" >}})
+
+```bash
+# The base image
+FROM outlinewiki/outline:version-0.44.0
+
+# Run the memory intensive build on an instance with 4 GB of memory (M)
+RUN yarn install && yarn build
+
+# Start the app on a smaller instance (nano)
+CMD yarn start
+```
+
+### TCP support
+
+Clever Cloud enables you to use TCP over Docker applications using using the environment variable `CC_DOCKER_EXPOSED_TCP_PORT=<port>`. Refer to the documentation page to know how to create [TCP redirections]({{< ref "doc/administrate/tcp-redirections.md" >}}].
```

Comment:
Hyperlink seems to be malformed
```suggestion
Clever Cloud enables you to use TCP over Docker applications using using the environment variable `CC_DOCKER_EXPOSED_TCP_PORT=<port>`. Refer to the documentation page to know how to create [TCP redirections]({{< ref "doc/administrate/tcp-redirections.md" >}}).
```

Reply:
This way of inserting links is useless, we can use the Markdown way: 
 
```suggestion
Clever Cloud enables you to use TCP over Docker applications using using the environment variable `CC_DOCKER_EXPOSED_TCP_PORT=<port>`. Refer to the documentation page to know how to create [TCP redirections](/doc/administrate/tcp-redirections).
```